src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -1280,12 +1280,17 @@ internal static BoundExpression NullOrDefault(TypeSymbol typeSymbol, SyntaxNode
                 case RefKind.Out:
                     refKind = RefKind.Ref;
                     break;
+
+                // "in" arguments are spilled by-value, while "strict-in" arguments are
```

Comment:
Can we use the terms "implicit in" and "explicit in". I think it's a bit clearer.